tools/Google.Cloud.Tools.DocUploader/Compress.cs

```diff
@@ -0,0 +1,71 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License").
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at 
+//
+// https://www.apache.org/licenses/LICENSE-2.0 
+//
+// Unless required by applicable law or agreed to in writing, software 
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and 
+// limitations under the License.
+
+using System.Diagnostics;
+
+namespace Google.Cloud.Tools.DocUploader;
+
+/// <summary>
+/// Helper class to perform the .tar.gz compression using powershell.
+/// </summary>
+public class Tar
+{
+    public static void Compress(string directory, string destination)
```

Comment:
Drive-by comment to start with: let's not run PowerShell and tar... it should be easy enough to create the tar file using SharpCompress.

Reply:
done